### DISCUSSION TITLE
cdn

### DISCUSSION THREAD

**Discussion by shuvvan** [hiddify/Hiddify-Manager#510] [Q&A سوالات رایج]
سلام 

من تمام موضوعات داخل کانال و گروه و اینجا رو گشتم یه آموزش ساده و چند خطی برای قرار دادن پنل پشت cdn نبود! تو گروه سوال کردم بلاک شدم

خواهشا یکی به زبان ساده بگه چطوری ببریم پشت کلودفلر! برای ما مبتدیا سخته به خدا بلد نیستیم !
آیا ای پی دومی نیازه؟

**Comment by zZedix** [comment]
سلام دوست عزیز !
برا استفاده از سرویس CDN شما باید از یکی از سایت هایی که این امکان رو ارایه میدن استفاده کنی مثل Cloudflare که بهترینشونه
با ثبت دامنه داخل این سایت زمان ساخت رکورد یا هم ساب دامنه یه گزینه داره با آیکون ابر به اسم Proxy اون رو برای رکورد تون روشن کنی اون ساب دامنه میره پشت cdn
حالا همون ساب دامنه رو با حالت CDN میای توی دامنه های هیدیفای اضافه میکنی !
اینم بگم توی تنظیمات cloudflare حتما SSL/TLS رو بذار روی FULL

**Reply by shuvvan** [reply]
تشکر مرسی
الان اون کارو انجام دادم و گذاشتم رو autocdn
ولی یه چندتا مشکل هست 
اول اینکه @تو اندروید باید یه چندبار رفرش بزنی تا یه آی پی خوب گیر بیاد !
دوم تو آیفون خیلی مشکله چون بروزرسانی نداره و باید هربار اضافه کنی لینکو بعد هم اونم یه چندباری باید بزنی !

یه راهی برای این هست؟

البته می تونم یه آیپی از اروان یا آسیا ایران تهیه کرد و انتقال بدیم به اون سرور که اینم هزینه زیاد داره یعنی هز 3 روز 1بار من 100 ت اسیا رو شارژ میکنم

**Reply by nikolay167** [reply]
Will you be interested if we will provide a CDN ?
